tuxemon/event/conditions/variable_is.py

```diff
@@ -49,17 +49,17 @@ def test(self, session: Session, condition: MapCondition) -> bool:
         operand2 = number_or_variable(session, condition.parameters[2])
 
         # Check if the condition is true
-        if operation == "==":
```

Comment:
I don't want to remove these, as it can be annoying for people who have work in progress stuff that is using it. There have been a couple of complaints about us breaking things before.

You can add support for the new comparisons, and change the existing comparisons to use the word versions, but let's leave these in for now.

Reply:
@kerizane by the way this:

> There have been a couple of complaints about us breaking things before.

Kinda annoying. So generic that makes me doubt about everything I have done.
No indication if recent complaining or regarding what.
No one replying to any PR telling: "this could break my progress".
I don't know what to think honestly. Disappointed.

Reply:
Please don't be put off by this, you have done so much great work. I'm only aware of two "complaints", and that's over-stating it, really.

6 - 12 months ago, somebody (maybe @Qiangong2 ?) updated their repo, and had a few issues due to some changes you'd made. A month ago they posted a warning to other users saying that "dialog" crashes the game.

That's it.